src/components/row.js

```diff
@@ -8,6 +8,19 @@ import {
 import {formatDistance} from 'date-fns';
 import {Link} from 'react-router-dom';
 
+function lastUpdate(props) {
+  const {lastupdatedtime: lastUpdatedTime} = props.state;
```

Comment:
I think it makes sense to directly input props.lastupdatedtime. No point in passing the whole thing

Reply:
@shuklaayush right. Done.